examples/ios-beta/main.rs

```diff
@@ -36,6 +38,11 @@ impl ViewDelegate for RootView {
         self.blue.set_background_color(Color::SystemBlue);
         view.add_subview(&self.blue);
 
+        let image_bytes = include_bytes!("../../test-data/favicon.ico");
+        self.image = ImageView::new();
+        self.image.set_image(&Image::with_data(image_bytes));
+        view.add_subview(&self.image);
```

Comment:
I don't love that this is the way the iOS simulator app looks but it does show that it graphically works.

<img width="370" alt="image" src="https://user-images.githubusercontent.com/1163510/185777434-f10d9065-d957-494a-8d53-ee6d1362ccc4.png">

Reply:
Ha, I can write up some autolayout hell for this to make it look "nicer" once this is merged. I'd really like to get labels and listview support in, so it'd be cool to get a true "kitchen sink" iOS demo at some point here.

Reply:
Yeah. It'd be nice if it could be use similar to the other views of `LayoutConstraint::activate`. I agree that it should be out of the scope of this PR though.